components/kubeflow-notebooks/Makefile

```diff
@@ -18,16 +20,22 @@ build:
 	@echo "build not implemented"
 .PHONY: build
 
-dist:
+dist: manifests
 	mkdir -p dist
 	cp ../kubeflow-pipeline/python/dist/fl_suite-${PIP_VERSION}-py3-none-any.whl dist/
 	sed -e "s/\\\$$\\\$$container-tag\\\$$\\\$$/${PIP_VERSION}/g" Dockerfile.tpl > dist/Dockerfile
 	sed -e "s/\\\$$\\\$$container-tag\\\$$\\\$$/${PIP_VERSION}/g" requirements.tpl.txt > dist/requirements.txt
-	DOCKER_BUILDKIT=1 docker build -t ${MAKEVAR_REGISTRY}/katulu-fl-jupyter-full:${CONTAINER_TAG} dist
+	DOCKER_BUILDKIT=1 docker build -t ${IMAGE_URL} dist
 .PHONY: dist
 
+manifests:
+	@cd kustomize/base/ && \
```

Comment:
Let's have a component level kustomize template instead to follow the general component layout